services/harborregistry/harborregistry.yml

```diff
@@ -0,0 +1,255 @@
+  metadata:
+    name: ${SERVICE_NAME} # harborregistry
+    labels:
+      service: ${SERVICE_NAME}
+      branch: ${SAFE_BRANCH}
+      project: ${SAFE_PROJECT}
+  type: Opaque
+  data:
+    REGISTRY_REDIS_PASSWORD: ""
+    #REGISTRY_STORAGE_S3_ACCESSKEY: "" # Will be injected via GraphiQL at build time
+    #REGISTRY_STORAGE_S3_SECRETKEY: "" # Will be injected via GraphiQL at build time
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: ${SERVICE_NAME} # harborregistry
+    labels:
+      service: ${SERVICE_NAME}
+      branch: ${SAFE_BRANCH}
+      project: ${SAFE_PROJECT}
+  data:
+    config.yml: |
+      version: 0.1
+      log:
+        level: error
+        fields:
+          service: registry
+      storage:
+        s3:
+          region: $STORAGE_AMAZON_REGION
+          bucket: $STORAGE_AMAZON_BUCKET
+        cache:
+          layerinfo: redis
+        maintenance:
+          uploadpurging:
+            enabled: false
+        delete:
+          enabled: true
+        redirect:
+          disable: false
+      redis:
+        addr: "harbor-redis:6379"
+        db: 2
+      http:
+        addr: :5000
+        relativeurls: true
+        # set via environment variable
+        # secret: placeholder
+        debug:
+          addr: localhost:5001
+      auth:
+        token:
+          issuer: harbor-token-issuer
+          realm: "$HARBOR_NGINX_ENDPOINT/service/token"
+          rootcertbundle: /etc/registry/root.crt
+          service: harbor-registry
+      validation:
+        disabled: true
+      notifications:
+        endpoints:
+          - name: harbor
+            disabled: false
+            url: http://harbor-core/service/notifications
+            timeout: 3000ms
+            threshold: 5
+            backoff: 1s
+    ctl-config.yml: |
+      ---
+      protocol: "http"
+      port: 8080
+      log_level: error
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: ${SERVICE_NAME} # harborregistry
+    labels:
+      service: ${SERVICE_NAME}
+      branch: ${SAFE_BRANCH}
+      project: ${SAFE_PROJECT}
+  spec:
+    ports:
+      - name: registry
+        port: 5000
+      - name: controller
+        port: 8080
+    selector:
+      service: ${SERVICE_NAME}
+      branch: ${SAFE_BRANCH}
+      project: ${SAFE_PROJECT}
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: ${SERVICE_NAME} # harborregistry
+    labels:
+      service: ${SERVICE_NAME}
+      branch: ${SAFE_BRANCH}
+      project: ${SAFE_PROJECT}
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        service: ${SERVICE_NAME}
+        branch: ${SAFE_BRANCH}
+        project: ${SAFE_PROJECT}
+    template:
+      metadata:
+        labels:
+          service: ${SERVICE_NAME}
+          branch: ${SAFE_BRANCH}
+          project: ${SAFE_PROJECT}
+      spec:
+        containers:
+        - name: ${HARBORREGISTRY_SERVICE_NAME}
+          image: ${HARBORREGISTRY_SERVICE_IMAGE} # cdchris12/harbor-registry:dev1.10
```

Comment:
this image comment is not correct anymore?

Reply:
Comment removed.